### PR TITLE
Help update2

### PR DESCRIPTION
**Pull request by alchemistmatt**
Fixes based on comments in https://github.com/AddstarMC/Prism-Bukkit/pull/142

### CODE REVIEW

File: src/main/java/me/botsko/prism/commands/FlagsCommand.java [hunk 1]
```diff
@@ -36,7 +36,7 @@ private void help(CommandSender sender) {
 		sender.sendMessage(Prism.messenger.playerMsg(
 				ChatColor.GRAY + "Flags control how Prism applies a rollback/restore, or formats lookup results."));
 		sender.sendMessage(Prism.messenger
-				.playerMsg(ChatColor.GRAY + "Use them after parameters, like /pr l p:<player_name> -extended"));
+				.playerMsg(ChatColor.GRAY + "Use them after parameters, like /pr l p:[player] -extended"));
```
Comment (addstar) [inline src/main/java/me/botsko/prism/commands/FlagsCommand.java]: Why the change? Standard syntax would be `<player>` for required values. Typically `[player]` indicates optional values.

Comment (alchemistmatt) [reply src/main/java/me/botsko/prism/commands/FlagsCommand.java]: I searched all of the files, and there is a mix of `[player]`
* https://github.com/AddstarMC/Prism-Bukkit/blob/master/src/main/java/me/botsko/prism/commands/ParamsCommand.java#L41
* https://github.com/AddstarMC/Prism-Bukkit/blob/master/src/main/java/me/botsko/prism/commands/ParamsCommand.java#L60

and  `<player>`
* https://github.com/AddstarMC/Prism-Bukkit/blob/master/src/main/java/me/botsko/prism/commands/ParamsCommand.java#L40
* https://github.com/AddstarMC/Prism-Bukkit/blob/master/src/main/java/me/botsko/prism/commands/FlagsCommand.java#L39

I'd be happy though, to switch them all to `<player>` -- I don't care either way.  I just noticed that the previous commit switched this one to `<player>`

Comment (alchemistmatt) [reply src/main/java/me/botsko/prism/commands/FlagsCommand.java]: Oh! I just realized your delineation between standard and optional.  Perhaps that's why there is a mix.  So, I'll back out that change to `[player]` and instead change
` Base the radius around another place, like r:[player]:20 or r:20,35,10:5 (x,y,z)`
to
` Base the radius around another place, like r:player:20 or r:20,35,10:5 (x,y,z)`

Comment (Narimm) [reply src/main/java/me/botsko/prism/commands/FlagsCommand.java]: Yeah Viv used [player] - I had used <player_name> as it was more standard in mc help

Comment (addstar) [reply src/main/java/me/botsko/prism/commands/FlagsCommand.java]: I'm not too fussed either, just thought it was worth pointing out the standard command line convention (in general, not with Prism) is to show required fields with `<player>` and optional with `[player]` :)

File: src/main/java/me/botsko/prism/commands/ParamsCommand.java [hunk 2]
```diff
@@ -38,7 +38,7 @@ private void help(CommandSender sender) {
 				+ " Force a worldwide search, for lookups only (unless configured for rollbacks)."));
 		sender.sendMessage(
 				Prism.messenger.playerMsg(ChatColor.LIGHT_PURPLE + "r:<player|x,y,z>:[radius]" + ChatColor.WHITE
-						+ " Base the radius around another place, like r:[player]:20 or r:20,35,10:5 (x,y,z)"));
+						+ " Base the radius around another place, like r:player:20 or r:20,35,10:5 (x,y,z)"));
```
Comment (Narimm) [inline src/main/java/me/botsko/prism/commands/ParamsCommand.java]: this should still use <player> to indicate the sender needs to replace that variable

**Review by Narimm — CHANGES_REQUESTED** [review]

File: src/main/java/me/botsko/prism/commands/ParamsCommand.java (hunk 2 above)
Comment (Narimm) [reply src/main/java/me/botsko/prism/commands/ParamsCommand.java]: Sorry markdown filtered my `<`

Comment (Narimm) [reply src/main/java/me/botsko/prism/commands/ParamsCommand.java]: ie use `<player>`

Comment (addstar) [reply src/main/java/me/botsko/prism/commands/ParamsCommand.java]: I'd agree with that.. you know someone is literally going to type `r:player:20` :facepalm:

**Review by addstar — DISMISSED** [review]

**Review by Narimm — APPROVED** [review]
